src/locale/locales/de/messages.po

```diff
@@ -2373,35 +2207,35 @@ msgstr "Journalismus"
 
 #: src/components/moderation/LabelsOnMe.tsx:59
 msgid "label has been placed on this {labelTarget}"
-msgstr ""
+msgstr "Etikett wurde auf diesem {labelTarget} platziert"
```

Comment:
I think we can keep `label` as `Label` everywhere.

Reply:
OK I'll try and do some changes to all the strings affected.